roles/openshift_node/defaults/main.yml

```diff
@@ -82,12 +82,16 @@ openshift_node_kubelet_args_dict:
 
 l_node_kubelet_args_default: "{{ openshift_node_kubelet_args_dict[openshift_cloudprovider_kind | default('undefined')] }}"
 
+l_node_kubelet_feature_flag_dict:
```

Comment:
No need to stick this in it's own dictionary and also modify l2 dict below.

Just put this feature-gates in the openshift_node_kubelet_args_dict.  Or, we should refactor the common bits out of openshift_node_kubelet_args_dict.